input/README.md

```diff
@@ -39,6 +39,22 @@ A	B	0.98
 B	C	0.77 
 ```
 
+### Gold Standard
+#### Nodes
+Gold standard node files are  text files and do not include a header row.
+
+Each row in the file represents a single node identifier. The file is structured as a single column with one node per line. These nodes typically correspond to gene or protein identifiers that are relevant to the biological pathway of interest.
+
+For example:
+```
+A
+B
+C
+```
+
+#### Edges
+TBD
```

Comment:
@sumedhars once you have the EGFR pathway edge gold standard ready, we can update this documentation in a follow up pull request.